templates/1_set_up_toffy.ipynb

```diff
@@ -15,9 +15,10 @@
    "id": "e36293c5-aa89-4029-a3fa-e8ea841bb8b5",
```

Comment:
<p>Can't this be done programmatically here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Sweeps aren't created in their own folder, they're just put into the main /Data folder. So you need to separately identify each FOV from the sweep, which are given the generic names. This was initially what that <code>find_detector_sweeps</code> function was for, but then Erin had a couple sweeps where FOVs were missing, so it gave an error. </p><p> </p><p>We could change it so that it would give a warning when an FOV is missing, rather than an error, and then ask people to list the first FOV and last FOV of their sweep and it would find the rest, but at that point it started to feel like the solution was almost as complicated as the problem. Up to you though, it would be an easy change </p>